embedded-voltdb/src/test/resources/application-disabled.properties

```diff
@@ -0,0 +1,2 @@
+spring.datasource.driver-class-name=org.voltdb.jdbc.Driver
```

Comment:
these properties are common for both profiles, put them in ```application.properties```

Reply:
Not sure if it makes sense to add 1 config file for 1 line

Reply:
you would remove duplications - it's always good.